### DISCUSSION TITLE
技术文档=>后台模拟器

### DISCUSSION THREAD

**Discussion by zcyhZhang** [dotnetcore/BootstrapBlazor#4199] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

后台模拟器
黑色主题时进入模拟器页面出现部分留白情况

### Expected Behavior

留白部分也跟随主题颜色而变化

### Steps To Reproduce

![微信截图_20240903144215](https://github.com/user-attachments/assets/d488b2d5-fdef-4ce1-8cfd-e81a59197e43)

```[tasklist]
### Tasks
```

**Comment by bb-auto** [comment]
@zcyhZhang Thank you for reporting. We will give triage later.

**Comment by zcyhZhang** [comment]
https://www.blazor.zone/layout-page

**Reply by ArgoZhang** [reply]
这个问题不是 BUG 是演示网站示例没有适配暗黑模式导致的，可以后续更新一下官网示例网站
